### ISSUE TITLE
Navigation: Cannot tab through links in dropdown

### ISSUE THREAD

**Issue by benjamincharity** [GetTerminus/terminus-ui#811]
A user should be able to tab through all navigation links:

link > link > link > dropdown trigger > dropdown link 1 > dropdown link 2
